### DISCUSSION TITLE
Setting/General Not there?

### DISCUSSION THREAD

**Discussion by whazza1983** [alexjustesen/speedtest-tracker#1718] [Q&A]
Hello, I've seen in many videos that there is a setting/general. 
But it's not there for me! Has it been removed?

![Setting-General](https://github.com/user-attachments/assets/1e2198a4-978c-4718-a3c1-d6e9508e5577)

`services:
    speedtest-tracker:
	image: lscr.io/linuxserver/speedtest-tracker:latest
        container_name: speedtest-tracker
        environment:
            - PUID=1000
            - PGID=1000
	        - TZ=etc/UTC
            - APP_KEY=
            - DB_CONNECTION=sqlite
            - SPEEDTEST_SCHEDULE=
            - SPEEDTEST_SERVERS=
            - DB_HOST= #optional
            - DB_PORT= #optional
            - DB_DATABASE= #optional
            - DB_USERNAME= #optional
            - DB_PASSWORD= #optional
            - DISPLAY_TIMEZONE=Etc/UTC #optional
            - PRUNE_RESULTS_OLDER_THAN=0 #optional
        volumes:
            - /portainer/Speedtest/config:/config
	    ports:
            - 8383:80
        restart: unless-stopped`

**Comment by svenvg93** [comment]
Hi @whazza1983, 

Yess these have been removed in a previous release. Please check out https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.20.0 for more information.

**Comment by whazza1983** [comment]
Thanks for the info.
